static/css/section/_mobile.scss

```diff
@@ -1,40 +1,84 @@
-.mobile-features {
-  .row-hero {
-    overflow: hidden;
-
-    @media only screen and (min-width : $breakpoint-medium) {
+.mobile- {
```

Comment:
I discussed it with @anthonydillon, he agreed it's better not to have partial class names to reduce a tiny amount of repetition in this way.